docs/runbooks/KubevirtHyperconvergedClusterOperatorSingleStackIPv6.md

```diff
@@ -0,0 +1,23 @@
+# KubevirtHyperconvergedClusterOperatorSingleStackIPv6
```

Comment:
Just a comment that the runbook name is very long. :-)

Reply:
+1 I think it would be an excellent idea to introduce a set of standards or conventions for naming runbooks. I see that there are other "KubevirtHyperconvergedOperator etc" runbooks. The names are so long that they do not fit on the TOC of the docs. :-(

![image](https://github.com/kubevirt/monitoring/assets/31725355/d2a770b8-9622-4586-a46b-ad968fad0339)

Reply:
Done. I've also replaced "OpenShift Virtualization" with "KubeVirt Hyperconverged".

Reply:
@apinnick Ill open a bug to update the alerts names